MaeParser.cpp

```diff
@@ -492,8 +492,8 @@ void IndexedBlockBuffer::value(Buffer& buffer)
                     break;
                 }
                 ++buffer.current;
-                m_tokens_list.setTokenIndices(save - buffer.begin,
-                                              buffer.current - buffer.begin);
+                m_tokens_list.setTokenIndices(save - buffer.begin + 1,
```

Comment:
simply trimming by +1/-1 will not handle escaped quotes and backslashes

Reply:
Dealing with quoted strings and slashes will definitely be a more substantial change we have to think about, as now we're talking about actually modifying the stored memory, which is something we try very hard not to do for performance reasons.  I'm not even sure if there *is* a way to do that without some substantial rewriting.